Marlin/src/lcd/ultralcd.h

```diff
@@ -34,6 +34,9 @@
 
 // I2C buttons must be read in the main thread
 #define HAS_SLOW_BUTTONS EITHER(LCD_I2C_VIKI, LCD_I2C_PANELOLU2)
+#ifdef HAS_SLOW_BUTTONS
+	extern millis_t next_button_update_ms;
+#endif
```

Comment:
`MarlinUI` already has a data member called `next_button_update_ms`. What is this supposed to fix?

Reply:
I had the same question, but if it's not present and declared as `extern ` there is another module (dont remember, think it was ultralcd.cpp) throws error `next_button_update_ms not declared `  my guess is that it is compiling and linking in some weird order that ultralcd doesn't find the variable of `MarlinUI`, any help or hint would be appreciated